mm/kfence/core.c

```diff
@@ -559,12 +559,53 @@ static const struct file_operations objects_fops = {
 	.llseek = seq_lseek,
 };
 
+#define CRASH_BUF_SIZE 16
```

Comment:
Damn. I did `git checkout HEAD~1` to hide this patch, but apparently it didn't work.
Please disregard it.